src/nupic/frameworks/opf/common_models/cluster_params.py

```diff
@@ -86,9 +91,16 @@ def getScalarMetricWithTimeOfDayAnomalyParams(metricData,
     maxVal = minVal + 1
 
   # Load model parameters and update encoder params
-  paramFileRelativePath = os.path.join(
-    "anomaly_params_random_encoder",
-    "best_single_metric_anomaly_params.json")
+  if (tmImplementation is "cpp"):
```

Comment:
@BoltzmannBrain I just made the formatting changes but the PR was closed on Friday. Should it be re-opened?